libbeat/_meta/fields.ecs.yml

```diff
@@ -259,6 +259,11 @@
             description: >
               Region ISO code.
 
+          - name: name
```

Comment:
Why is there such a large diff? Did you replace the ECS yml? Please don't do this in this PR.